packages/blitz-auth/package.json

```diff
@@ -31,10 +31,12 @@
     "cookie-session": "2.0.0",
     "debug": "4.3.3",
     "http": "0.0.1-security",
+    "ioredis": "5.2.1",
```

Comment:
I think we should find a solution where we avoid adding Redis to the dependencies. Currently, everyone using blitz-auth will end up having ioredis and redis in their deps even if they aren't using it.

Reply:
Both of these dependencies can be added to devDependencies , taking into account the typescript definitions.

Maybe it is possible to provide an abstraction layer up to the user concrete implementation to decide which one to use?

Reply:
Maybe have `@blitzjs/auth-backend-redis`?

Reply:
> Maybe have `@blitzjs/auth-backend-redis`?

Yeah, I was thinking about `@blitzjs/auth/redis` or something. Not sure yet what would be best here

Reply:
Can we keep it in `@blitzjs/auth` but move redis stuff to devDeps and have users install it themselves?

That's what we do with Prisma and Zod, and it works well